src/js/components/events/events/components/event-card-new/EventCardNew.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+//import image from '../../../../../../img/eventplog-logo-name-inverted-medium'
+import styled from 'styled-components'
+import colors from '../../../../../../styles/theme/colors';
+import Button from '../../../../shared/button'
+import {Link} from 'react-router-dom'
+
+const StyledEventCardNew = styled.div`
+
+margin-left: 75px;
+margin-bottom: 15px;
+border-radius: .28571429rem;
+box-shadow: 0 1px 3px 0 #d4d4d5,0 0 0 1px #d4d4d5;
```

Comment:
Consider abstracting these colors to one file and then calling them in your css. This helps with uniformity of colors across all pages.